docs/api/scripts_gui_panels.md

```
@@ -0,0 +1,186 @@
+| Description | *No module description available.* |
+| Args | — |
+| Returns | — |
+
+
+## `scripts\gui\panels\action_panel`
+
+**🧠 Docstring Summary**
+
+| Section | Content |
+|---------|---------|
+| Description | action_panel.py
```

markdownlint-cli2 (0.17.2)

Table pipe style
Expected: leading_and_trailing; Actual: leading_only; Missing trailing pipe

(MD055, table-pipe-style)